SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTestFlowController.cpp

```diff
@@ -54,6 +59,59 @@ void ASpatialFunctionalTestFlowController::CrossServerSetWorkerId_Implementation
 	WorkerDefinition.Id = NewWorkerId;
 }
 
+void ASpatialFunctionalTestFlowController::AddEntityInterest_Implementation(const int64 ActorEntityId)
+{
+	USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(GetNetDriver());
+	if (SpatialNetDriver == nullptr || ActorEntityId == SpatialConstants::INVALID_ENTITY_ID)
+	{
+		ensureMsgf(ActorEntityId != SpatialConstants::INVALID_ENTITY_ID, TEXT("Trying to add interest over an invalid entity id"));
+		return;
+	}
+	
+	ULayeredLBStrategy* LayeredLBStrategy = Cast<ULayeredLBStrategy>(SpatialNetDriver->LoadBalanceStrategy);
+	USpatialFunctionalTestGridLBStrategy* TestGridLB = Cast<USpatialFunctionalTestGridLBStrategy>(LayeredLBStrategy->GetLBStrategyForVisualRendering());
+	if (TestGridLB != nullptr)
+	{
+		int NumEntities = TestGridLB->Entities.Num();
+		TestGridLB->Entities.AddUnique(ActorEntityId);
+		if( NumEntities != TestGridLB->Entities.Num())
+		{
+			SpatialNetDriver->Sender->UpdateServerWorkerEntityInterestAndPosition();
+		}
+	}
+	else
+	{
+		UE_LOG(LogSpatialGDKFunctionalTests, Error, TEXT("Trying to add interest without USpatialFunctionalTestGridLBStrategy"));
+	}
+}
+
+void ASpatialFunctionalTestFlowController::RemoveEntityInterest_Implementation(const int64 ActorEntityId)
+{
+	USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(GetNetDriver());
+	if (SpatialNetDriver == nullptr || ActorEntityId == SpatialConstants::INVALID_ENTITY_ID)
+	{
+		ensureMsgf(ActorEntityId != SpatialConstants::INVALID_ENTITY_ID, TEXT("Trying to remove interest over an invalid entity id"));
+		return;
+	}
+
+	ULayeredLBStrategy* LayeredLBStrategy = Cast<ULayeredLBStrategy>(SpatialNetDriver->LoadBalanceStrategy);
+	USpatialFunctionalTestGridLBStrategy* TestGridLB = Cast<USpatialFunctionalTestGridLBStrategy>(LayeredLBStrategy->GetLBStrategyForVisualRendering());
+
+	if (TestGridLB != nullptr)
+	{
+		int NumEntities = TestGridLB->Entities.Num();
+		TestGridLB->Entities.Remove(ActorEntityId);
+		if (NumEntities != TestGridLB->Entities.Num())
+		{
+			SpatialNetDriver->Sender->UpdateServerWorkerEntityInterestAndPosition();
+		}
+	}
+	else
+	{
+		UE_LOG(LogSpatialGDKFunctionalTests, Error, TEXT("Trying to add interest without USpatialFunctionalTestGridLBStrategy"));
```

Comment:
```suggestion
		UE_LOG(LogSpatialGDKFunctionalTests, Error, TEXT("Trying to remove interest without USpatialFunctionalTestGridLBStrategy"));
```
Also would recommend refactoring these two functions maybe if easy.

Reply:
Refactored